modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -96,8 +107,12 @@ spec:
         mode: 420
         overwrite: true
         path: /etc/clevis.json
+  kernelArguments:
+    - rd.neednet=1 <1>
 EOF
 ----
++
+<1> Required. Enables networking even when `netroot` utilities are not available so that the asymmetric keys can be obtained from the Tang server.
```

Comment:
This description has not been confirmed by our Engineering team, it is just my assumption as to what is happening here. Also, it provides a lot of detail that the reader might not have the necessary background for within the doc to understand fully.

I would suggest simplifying the statement just to say that the argument is required in order for the second boot to succeed. Also, you could expand the 'Required' statement a little so that it is more reader oriented. For example, the call out could be as follows:

"This kernel argument is required for the second boot to succeed."

Reply:
OK, cool. I wasn't sure how much detail to provide, so makes sense to simplify. I revised the annotated note. I like to front-load "Required" and "Optional" so a reader sees it right away when scanning. _(Side note: should we use a colon as we do for `Optional`? I'll stick with a period here for now as that's how I've written this in the past.)_